### ISSUE TITLE
ID SC2164 applies incorrectly to popd

### ISSUE THREAD

**Issue by systemmonkey42** [koalaman/shellcheck#1252]
#### For bugs
- Rule Id (if any, e.g. SC1000): SC2164
- My shellcheck version (`shellcheck --version` or "online"):  0.5.0
- [x] I tried on shellcheck.net and verified that this is still a problem on the latest commit
- [ ] It's not reproducible on shellcheck.net, but I think that's because it's an OS, configuration or encoding issue

#### For new checks and feature suggestions
- [ ] shellcheck.net (i.e. the latest commit) currently gives no useful warnings about this
- [ ] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

Related issues: #903 #863 

popd on BASH suppresses "cd"-like behaviour with -n.  SC2164 does not apply.

#### Here's a snippet or screenshot that shows the problem:

```sh
#!/bin/bash

popd -n +2
```

#### Here's what shellcheck currently says:
```
Line 3:
popd -n +2
^-- SC2164: Use 'cd ... || exit' or 'cd ... || return' in case cd fails.
```

#### Here's what I wanted or expected to see:

Nothing.

Kind Regards
